files/galaxy/tpv/tools.yml

```diff
@@ -509,6 +509,31 @@ tools:
       - if: input_size >= 20
         fail: Too much data, please don't use Spades for this
 
+  toolshed.g2.bx.psu.edu/repos/galaxy-australia/smudgeplot/smudgeplot/.*:
+    cores: 8
+    rules:
+      - if: input_size < 1
+        mem: 15
+      - if: 1 <= input_size < 5
+        mem: 75
+      - if: 5 <= input_size < 10
+        cores: 12
+        mem: 150
+      - if: 10 <= input_size < 15
+        cores: 12
+        mem: 225
+      - if: 15 <= input_size < 20
+        cores: 12
+        mem: 300
+      - if: 20 <= input_size < 25
+        mem: 375
+        cores: 16
+        scheduling:
+          prefer:
```

Comment:
If you really want to use the local cluster, make it a require:
Prefer doesn't really work atm

Reply:
`require` will only match if the destination has that tag
`prefer` can still match with every other destination, except for destinations that `refuse` that tag